conf/application.conf

```diff
@@ -53,6 +53,20 @@ microservice {
       host = localhost
       port = 9402
     }
+    des {
+      host=localhost
+      port=9904 #agent-stubs
+      authorization-token=secret
+      environment=test
+      retry {
```

Comment:
retry and cache configs might not be used in this repo right now

Reply:
👍

Reply:
👍